src/pages/components/sheet.mdx

```diff
@@ -0,0 +1,199 @@
+
+Sheets can be placed at the start or end of the container. End is the default placement.
+
+import SheetPlacement from '../../examples/chrome/SheetPlacement.tsx';
+import SheetPlacementCode from '!!raw-loader!../../examples/chrome/SheetPlacement.tsx';
+
+<CodeExample code={SheetPlacementCode}>
+  <SheetPlacement />
+</CodeExample>
+
+### Size
+
+Sheets default to 380px in width. You can customise the width based on your intended
+experience to any size using pixels or our grid system.
+
+import SheetSize from '../../examples/chrome/SheetSize.tsx';
+import SheetSizeCode from '!!raw-loader!../../examples/chrome/SheetSize.tsx';
+
+<CodeExample code={SheetSizeCode}>
+  <SheetSize />
+</CodeExample>
+
+## API
+
+The Sheet component follows this structure:
+
+<!-- prettier-ignore -->
+```jsx
+<Sheet>
+  <Sheet.Header>
+    <Sheet.Title>
+      {/* a text to label the sheet */}
+    </Sheet.Title>
+    <Sheet.Description>
+      {/* a text describing the sheet */}
+    </Sheet.Description>
+  </Sheet.Header>
+  
+  <Sheet.Body>
+    {/* main content of the Sheet */}
+  </Sheet.Body>
+
+  <Sheet.Footer>
+    <Sheet.FooterItem>
+      {/* items to set in the footer, such as buttons */}
+    </Sheet.FooterItem>
+  </Sheet.Footer>
+
+  <Sheet.Close />
+</Sheet>
+```
+
+### Sheet
+
+<Component components={props.data.mdx.components} componentName="sheet" />
+
+The Sheet component provides state and accessibility attributes to its children.
+
+<PropSheet components={props.data.mdx.components} componentName="sheet" />
+
+### Sheet.Body
+
+The Sheet Body component is placed in the [`Sheet`](#sheet) component.
+This component requires accessible functionality to make the
+element focusable, if keyboard scrolling is needed.
+
+<Component components={props.data.mdx.components} componentName="sheet.body" />
+
+<PropSheet components={props.data.mdx.components} componentName="sheet.body" />
+
+### Sheet.Title
+
+The Sheet Title labels the Sheet for assistive technology.
+This component is nested in the [`Sheet.Header`](#sheetheader) component.
+
+<Component components={props.data.mdx.components} componentName="sheet.title" />
+
+<PropSheet components={props.data.mdx.components} componentName="sheet.title" />
+
+### Sheet.Description
+
+The Sheet Description describes the Sheet for assistive technology.
+This component is nested in the [`Sheet.Header`](#sheetheader) component.
+
+<Component components={props.data.mdx.components} componentName="sheet.description" />
+
+<PropSheet components={props.data.mdx.components} componentName="sheet.description" />
+
+### Sheet.Header
+
+The Sheet Header component is placed in the [`Sheet`](#sheet) component.
+
+<Component components={props.data.mdx.components} componentName="sheet.header" />
+
+<PropSheet components={props.data.mdx.components} componentName="sheet.header" />
+
+### Sheet.Footer
+
+The Sheet Footer component is placed in the [`Sheet`](#sheet) component.
+The component can alternate to compact styling.
```

Comment:
Please reference other component documentation for use of active voice. Also, something about this "...can alternate" feels a bit off grammatically.

Reply:
Will update this and some of the text.